Release v1.20.0-rc (#2421)
googleforgames/agones: site/content/en/blog/releases/1.20.0-rc.md

---
title: "1.20.0 - Release Candidate"
linkTitle: "1.20.0-rc"
date: "2022-01-04"
---
This is the 1.20.0-rc release of Agones.

> This release provides new functionality to the the simple game server (now version 0.6), as well as bringing the grpc client and server to be the same version across the project.

Reviewer: nit: the the -> the

> This release provides new functionality to the the simple game server (now version 0.6), as well as bringing the grpc client and server to be the same version across the project.

Reviewer: ```suggestion
This release provides new functionality to the simple game server (now version 0.6), as well as bringing the grpc client and server to be the same version across the project.
```

---
title: "1.20.0 - Release Candidate"
linkTitle: "1.20.0-rc"
date: "2022-01-04"
---
This is the 1.20.0-rc release of Agones.

> This release provides new functionality to the the simple game server (now version 0,6), as well as bringing the grpc client and server to be the same version across the project.

Reviewer 2: ```suggestion
This release provides new functionality to the the simple game server (now version 0.6), as well as bringing the grpc client and server to be the same version across the project.
```

Check the <a href="https://github.com/googleforgames/agones/tree/release-1.20.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.

> **Implemented enhancements:**

Reviewer 2: ```suggestion
```

This is now here twice. Please remove this one.

---

> title: "1.20.0-Release Candidate"

Reviewer 2: ```suggestion
title: "1.20.0 - Release Candidate"
```

Reviewer 2: Same as: https://github.com/googleforgames/agones/blob/main/site/content/en/blog/releases/1.8.0-rc.md

linkTitle: "1.20.0-rc"
date: "2022-01-04"
---
This is the 1.20.0-rc release of Agones.

> This release brings us the simple game server version 0.6 and grpc client and server on same version

Reviewer 2: ```suggestion
This release provides new functionality to the the simple game server (now version 0,6), as well as bringing the grpc client and server to be the same version across the project.
```

Check the <a href="https://github.com/googleforgames/agones/tree/release-1.20.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.

**Implemented enhancements:**

Update the simple game server version to 0.6.[#2413](https://github.com/googleforgames/agones/pull/2413)

> Update the minikube driver flag name (--vm-driver is deprecated) [#2399]([url](https://github.com/googleforgames/agones/pull/2399))

Reviewer 2: ```suggestion
Update the minikube driver flag name (--vm-driver is deprecated) [#2399](https://github.com/googleforgames/agones/pull/2399)
```

This should fix it - just confirming locally.

> Update the simple game server version to 0.6.[#2413](https://github.com/googleforgames/agones/pull/2413)
> Update the minikube driver flag name (--vm-driver is deprecated) [#2399](https://github.com/googleforgames/agones/pull/2399)
> Unreal SDK misses "const" prefix for function parameters. [#2411](https://github.com/googleforgames/agones/issues/2411)
> TLS handshake error with multi-cluster setup [#2402](https://github.com/googleforgames/agones/issues/2402)
> Upgrade-gRPC-version [#2414](https://github.com/googleforgames/agones/pull/2414)
> Update the simple game server to move itself back to the Ready state after allocation [#2409](https://github.com/googleforgames/agones/pull/2409)

Reviewer 2: Sorry, I really should have looked at this closer, my apologies, but this is not following the process - this should be copy / pasted from the changelog.

```suggestion
**Implemented enhancements:**

- Update the simple game server to move itself back to the Ready state after allocation [\#2409](https://github.com/googleforgames/agones/pull/2409) ([roberthbailey](https://github.com/roberthbailey))
- Add a flag to simple-game-server to shutdown after a specified number of seconds [\#2407](https://github.com/googleforgames/agones/pull/2407) ([roberthbailey](https://github.com/roberthbailey))

```

If there are items we think should be marked as features, we should mark the issues as features during the issue/pr review when doing the release checklist.

This release document should match the CHANGELOG.

In the template we have:

```
{ insert enhancements from the changelog and/or security and breaking changes }
```

This should be a copy/paste operation from the generated markdown.

---

> title: "1.20.0-rc - simple game server version to 0.6"

Reviewer 3: This line should be `title: "1.20.0 - Release Candidate"`

Reviewer 2: Good catch, I missed that one!

linkTitle: "1.20.0-rc"
date: "2022-01-04"
---

> 

Reviewer 3: nit: No need for blank lines 6-7.

This is the 1.20.0-rc release of Agones.

> This release brings us  the simple game server version 0.6 and grpc client and server on same version

Reviewer 3: nit: two spaces between "us" and "the"

> Check the <a href="https://github.com/googleforgames/agones/tree/release-1.20.0-rc" >README</a> for details on features, installation and usage.

Reviewer 2: This will need `data-proofer-ignore` - see the template I linked in the last comment. (Since the branch doesn't exist yet).

**Implemented enhancements:**

> Update the simple game server version to 0.6.[ #2413](https://github.com/googleforgames/agones/pull/2413)

Reviewer 2: ```suggestion
Update the simple game server version to 0.6.[#2413](https://github.com/googleforgames/agones/pull/2413)
```
Space here too, not sure why.

Update the minikube driver flag name (--vm-driver is deprecated) [#2399]([url](https://github.com/googleforgames/agones/pull/2399))
Unreal SDK misses "const" prefix for function parameters. [#2411](https://github.com/googleforgames/agones/issues/2411)
TLS handshake error with multi-cluster setup [#2402](https://github.com/googleforgames/agones/issues/2402)

> Upgrade-gRPC-version [ #2414](https://github.com/googleforgames/agones/pull/2414)

Reviewer 2: ```suggestion
Upgrade-gRPC-version [#2414](https://github.com/googleforgames/agones/pull/2414)
```

(not sure why there is a space here?)

Update the simple game server to move itself back to the Ready state after allocation [#2409](https://github.com/googleforgames/agones/pull/2409)

Documentation: https://development.agones.dev/site/

> See <a href="https://github.com/googleforgames/agones/blob/release-1.20.0-rc/CHANGELOG.md" >CHANGELOG</a> for more details on changes.

Reviewer 2: This will need data-proofer-ignore - see the template I linked in the last comment. (Since the branch doesn't exist yet).

- [gcr.io/agones-images/agones-ping:1.20.0-rc](https://gcr.io/agones-images/agones-ping:1.20.0-rc)
- [gcr.io/agones-images/agones-allocator:1.20.0-rc](https://gcr.io/agones-images/agones-allocator:1.20.0-rc)
- [gcr.io/agones-images/autoscaler-webhook:0.4](https://gcr.io/agones-images/autoscaler-webhook:0.4)
- [gcr.io/agones-images/cpp-simple-server:0.13](https://gcr.io/agones-images/cpp-simple-server:0.13)
- [gcr.io/agones-images/crd-client:0.6](https://gcr.io/agones-images/crd-client:0.6)
- [gcr.io/agones-images/nodejs-simple-server:0.5](https://gcr.io/agones-images/nodejs-simple-server:0.5)
- [gcr.io/agones-images/rust-simple-server:0.9](https://gcr.io/agones-images/rust-simple-server:0.9)
- [gcr.io/agones-images/simple-game-server:0.6](https://gcr.io/agones-images/simple-game-server:0.6)
- [gcr.io/agones-images/supertuxkart-example:0.4](https://gcr.io/agones-images/supertuxkart-example:0.4)
- [gcr.io/agones-images/unity-simple-server:0.3](https://gcr.io/agones-images/unity-simple-server:0.3)
- [gcr.io/agones-images/xonotic-example:0.8](https://gcr.io/agones-images/xonotic-example:0.8)

Helm chart available with this release:

> - <a href="https://agones.dev/chart/stable/agones-1.20.0-rc.tgz" >

Reviewer 2: Pretty sure this also needs `data-proofer-ignore` - but check the template.
